linux_os/guide/system/accounts/accounts-physical/require_emergency_target_auth/oval/shared.xml

```diff
@@ -2,13 +2,25 @@
   <definition class="compliance" id="require_emergency_target_auth" version="1">
     {{{ oval_metadata("The requirement for a password to boot into emergency mode
       should be configured correctly.") }}}
-    <criteria operator="AND">
-      <criterion comment="Conditions are satisfied"
-      test_ref="test_require_emergency_service" />
-      <criterion test_ref="test_require_emergency_service_emergency_target" />
-      <criterion test_ref="test_no_custom_emergency_target" negate="true"/>
-      <criterion test_ref="test_no_custom_emergency_service" negate="true"/>
+{{% if 'sle' in product %}}
+    <criteria operator="OR">
+{{% endif %}}
+      <criteria operator="AND">
+        <criterion comment="Conditions are satisfied"
+          test_ref="test_require_emergency_service" />
+        <criterion test_ref="test_require_emergency_service_emergency_target" />
+        <criterion test_ref="test_no_custom_emergency_target" negate="true"/>
+        <criterion test_ref="test_no_custom_emergency_service" negate="true"/>
+      </criteria>
+{{% if 'sle' in product %}}
+      <criteria operator="OR">
```

Comment:
Could you move this block up? Then it is not necessary to use twice the same jinja2 conditional here.

Reply:
We need the same jinja condition to close the top-level `OR` criteria at line 5 I think anyways, no? It is just a matter of preference where to put the extra jinja condition at the start of the wrapping criteria, or at its closing, right?

Reply:
You are correct. I overlooked something. Nevermind. : )